circle.yml

```diff
@@ -242,6 +242,17 @@ commands:
           name: Generate documentation
           command: make javadoc-dokka
 
+  track-performance:
+    steps:
+      - run:
+          name: Track performance of the Navigation 1.0
+          command: |
+            if [ -n "${MOBILE_METRICS_TOKEN}" ]; then
+              if [[ $CIRCLE_BRANCH == master ]]; then
```

Comment:
If we're going to execute this command as part of `release-1_0-snapshot` this is not needed because that job is only executed in `master` https://github.com/mapbox/mapbox-navigation-android/blob/a942bccab327367650770b7002612f0e76a12f45/circle.yml#L14-L18 Look at https://github.com/mapbox/mapbox-navigation-android/pull/2520 https://github.com/mapbox/mapbox-navigation-android/blob/352ae201eeee4e4519ed58fff83fc04d03476d9b/circle.yml#L251-L255 We were following that approach.

Reply:
I'd like to keep this so that when we are publishing snapshots manually, with test builds, we're not triggering metrics runs unintentionally.